warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IsNotNullPruningVisitor.java

```diff
@@ -586,11 +559,6 @@ public Object visit(ASTMethodNode node, Object data) {
         return data;
     }
     
-    @Override
-    public Object visit(ASTSizeMethod node, Object data) {
```

Comment:
ASTSizeFunction

Reply:
There could be a null within a size function, so I think we should leave this as-is since the parent implementation for ASTSizeFunction will cover that case.